### ISSUE TITLE
FPS Unlocker (latest) doesn't work for some reason

### ISSUE THREAD

**Issue by kastentop2005** [34736384/genshin-fps-unlock#335]
Device: ASUS TUF F17 2023
OS: Windows 11 23H2

Tried editing config and game settings, no effect

**Comment by jonokhor** [comment]
Yeah, I find that it is not working for some reason too. That is, the FPS caps at 60 and cannot go higher.

**Comment by gdwneo** [comment]
it happens to me after I upgraded nvidia driver
